bindata/manifests/operator-webhook/002-rbac.yaml

```diff
@@ -32,6 +34,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: operator-webhook-role-binding
+  labels:
+    created-by: {{ include "sriov-network-operator.fullname" . }}  
```

Comment:
I'm not an expert of Helm, but I think `sriov-network-operator.fullname` has meaning inside the `deployment/` folder only. `bindata/` folder is rendered by the operator. Am I missing something?